stl/src/vector_algorithms.cpp

```diff
@@ -1309,32 +1339,6 @@ namespace {
                         break; // No wrapping, so it was the only portion

```

Comment:
The `continue;` on right line 1337 is no longer necessary. I think we should drop it, following the usual guideline that avoiding unnecessary code highlights the importance of the remaining code (i.e. every `continue;` in the codebase should actually bypass code). This is the only `continue;` in this file.